docs/book/src/quick-start.md

```diff
@@ -24,60 +34,53 @@ sudo mv /tmp/kubebuilder_2.0.0_${os}_${arch} /usr/local/kubebuilder
 export PATH=$PATH:/usr/local/kubebuilder/bin
 ```
 
-You can also install a KubeBuilder master snapshot from
-`https://go.kubebuilder.io/dl/latest/${os}/${arch}`.
+**NOTE:** Also, you can install a master snapshot from `https://go.kubebuilder.io/dl/latest/${os}/${arch}`.
 
-Install [kustomize](https://sigs.k8s.io/kustomize/docs/INSTALL.md) v3.1.0+
+* Install [kustomize](https://sigs.k8s.io/kustomize/docs/INSTALL.md) v3.1.0+
```

Comment:
why did you change all the bullet points below?

Reply:
To make easier see/identify that has more than one step to follow up and this part does not be skipped by mistake.

Reply:
After changes, it makes no longer sense.  Tks

Reply:
Solved